docs/source/error-codes/self-service/index.rst

```diff
@@ -11,81 +11,185 @@ Self-Service Error Codes (Experimental)
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|TransactionService.getTransactionTreesSource      |NOT_FOUND                        |FAILED_PRECONDITION                    |                                                                                                                         |LEDGER_ID_MISMATCH                           |
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|TransactionService.getTransactions                |INVALID_ARGUMENT                 |FAILED_PRECONDITION                    |FAILED_PRECONDITION can now be returned when the supplied offset was out of range.                                       |OFFSET_OUT_OF_RANGE                          |
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|TransactionService.getTransactions                |NOT_FOUND                        |FAILED_PRECONDITION                    |                                                                                                                         |LEDGER_ID_MISMATCH                           |
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|TransactionService.getTransactions                |NOT_FOUND                        |FAILED_PRECONDITION                    |FAILED_PRECONDITION can now be returned when attempting to access the data that has already been pruned.                 |PARTICIPANT_PRUNED_DATA_ACCESSED             |
++--------------------------------------------------+---------------------------------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------+---------------------------------------------+
+Sandbox Classic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following gRPC error codes have changed for submission rejections in Sandbox classic.
```

Comment:
Aren't we simply calling this component 'Sandbox' in Daml Connect?

Reply:
Right 👍

Reply:
Renamed it as Sandbox (classic) in order not to conflict with Daml Sandbox